assets/extensions/extensions.scss

```diff
@@ -252,13 +254,15 @@ $wordpress-break: 961px;
 	&__grid-list {
 		margin: 0;
 		padding: 0;
+		align-items: stretch;
```

Comment:
Removing the previously commented `align-items: flex-start;`, it wouldn't be necessary.